astraguard/swarm/registry.py

```
@@ -0,0 +1,386 @@
+                if other_peers:
+                    fanout_size = min(GOSSIP_FANOUT, len(other_peers))
+                    targets = random.sample(other_peers, fanout_size)
+                    
+                    for target in targets:
+                        try:
+                            await self.bus.publish(
+                                topic=f"satellite/hello/{sender_agent_id.id}",
+                                payload=payload,
+                                qos_level=0  # Fire-and-forget
+                            )
+                        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)